static/noto.css

```diff
@@ -0,0 +1,93 @@
+/* Arabic */
+@font-face {
+    font-family: "Noto Serif";
+    font-weight: 400;
+    src: url("fonts/arabic/NotoNaskhArabic-Regular.ttf");
+    unicode-range: U+0600-06FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 400;
+    src: url("fonts/arabic/NotoSansArabic-Regular.ttf");
+    unicode-range: U+0600-06FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 700;
+    src: url("fonts/arabic/NotoSansArabic-Bold.ttf");
+    unicode-range: U+0600-06FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 900;
+    src: url("fonts/arabic/NotoSansArabic-Black.ttf");
+    unicode-range: U+0600-06FF;
+}
+
+/* Hebrew */
+@font-face {
+    font-family: "Noto Serif";
+    font-weight: 400;
+    src: url("fonts/hebrew/NotoSerifHebrew-Regular.ttf");
+    unicode-range: U+0590-05FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 400;
+    src: url("fonts/hebrew/NotoSansHebrew-Regular.ttf");
+    unicode-range: U+0590-05FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 700;
+    src: url("fonts/hebrew/NotoSansHebrew-Bold.ttf");
+    unicode-range: U+0590-05FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 900;
+    src: url("fonts/hebrew/NotoSansHebrew-Black.ttf");
+    unicode-range: U+0590-05FF;
+}
+
+/* Cyrillic */
+@font-face {
+    font-family: "Noto Serif";
+    font-weight: 400;
+    src: url("fonts/NotoSerif-Regular.ttf");
+    unicode-range: U+0400-04FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 400;
+    src: url("fonts/NotoSans-Regular.ttf");
+    unicode-range: U+0400-04FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 700;
+    src: url("fonts/NotoSans-Bold.ttf");
+    unicode-range: U+0400-04FF;
+}
+
+@font-face {
+    font-family: "Noto Sans";
+    font-weight: 900;
+    src: url("fonts/NotoSans-Black.ttf");
+    unicode-range: U+0400-04FF;
+}
+
+/* Cree */
```

Comment:
How were these languages selected? We don't have any Cree on our site (I don't think) but we do have Japanese, for example (there is also a Noto Sans for Chinese but it appears to be massive).

Reply:
Cree was for fun.  I'll replace it with something else. I'm deferring CJK languages for now due to the size of the font files.